Code/GraphMol/ChemReactions/Wrap/rdChemReactions.cpp

```diff
@@ -907,6 +907,29 @@ of the replacements argument.",
               (python::arg("reaction"), python::arg("params")),
               "construct a reaction SMILES string for a ChemicalReaction");
 
+  python::def("ReactionToCXSmarts",
+              (std::string(*)(const RDKit::ChemicalReaction &))
+                  RDKit::ChemicalReactionToRxnCXSmarts,
+              (python::arg("reaction")),
+              "construct a reaction SMARTS string for a ChemicalReaction");
+  python::def("ReactionToCXSmiles",
+              (std::string(*)(const RDKit::ChemicalReaction &,
+                              bool))RDKit::ChemicalReactionToRxnCXSmiles,
+              (python::arg("reaction"), python::arg("canonical") = true),
+              "construct a reaction SMILES string for a ChemicalReaction");
+  python::def("ReactionToCXSmarts",
+              (std::string(*)(const RDKit::ChemicalReaction &,
+                              const RDKit::SmilesWriteParams &))
+                  RDKit::ChemicalReactionToRxnCXSmarts,
+              (python::arg("reaction"), python::arg("params")),
+              "construct a reaction CXSMARTS string for a ChemicalReaction");
+  python::def("ReactionToCXSmiles",
+              (std::string(*)(const RDKit::ChemicalReaction &,
+                              const RDKit::SmilesWriteParams &))
+                  RDKit::ChemicalReactionToRxnCXSmiles,
+              (python::arg("reaction"), python::arg("params")),
+              "construct a reaction CXSMILES string for a ChemicalReaction");
+
```

Comment:
So currently, we aren't passing in the `RDKit::SmilesWrite::CXSmilesFields` flag to determine what to write out in the CX extensions, and I think we should. Here is where `MolToCXSmiles` does it: https://github.com/rdkit/rdkit/blob/7d2598267acc80b4aee03ea20c4297ae4e1c9ea3/Code/GraphMol/SmilesParse/SmilesWrite.h#L280

I think this will involve figuring out how to pass the flag into the `getCXExtensions` function you wrote and overloading `ChemicalReactionToRxnCXSmarts` and `ChemicalReactionToRxnCXSmiles` to include a `flags` argument.